ci-operator/step-registry/dpf/hypervisor/load-kubeconfig/dpf-hypervisor-load-kubeconfig-commands.sh

```
@@ -0,0 +1,77 @@
+# Test SSH connection
+echo "Testing SSH connection to ${REMOTE_HOST}..."
+if ssh ${SSH_OPTS} root@${REMOTE_HOST} echo 'SSH connection successful'; then
+    echo "SSH setup complete and tested successfully"
+else
+    echo "ERROR: Failed to connect to hypervisor ${REMOTE_HOST}"
+    exit 1
+fi
+
+# Find the last DPF openshift-dpf install dir on the hypervisor
+echo "=== Locating last openshift-dpf install dir on hypervisor ==="
+scp ${SSH_OPTS} root@${REMOTE_HOST}:${REMOTE_LAST_OPENSHIFT_DPF_DIR_LOCATION} /tmp
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)